test/src/Makefile

```diff
@@ -25,7 +25,7 @@ fmt:
 	$(V) fmt -w r2r.v
 
 v:
-	git clone https://github.com/vlang/v
+	git clone --depth 1 https://github.com/vlang/v
```

Comment:
that will break the git reset --hard line

Reply:
heheh same time!

Reply:
sorry fixed in fd0abf92028ff1b7336eb328a11925ad5d0e7531